src/Styleguide/Pages/Artist/index.js

```diff
@@ -1,62 +1,74 @@
+import { Theme, themeProps, injectGlobalCSS } from "@artsy/palette"
 import React from "react"
-import { Tabs, Tab } from "Styleguide/Components/Tabs"
+import { Footer } from "Styleguide/Components/Footer"
+import { Tab, Tabs } from "Styleguide/Components/Tabs"
+import { GlobalStyles } from "Styleguide/Elements/GlobalStyles"
 import { Col, Grid, Row } from "Styleguide/Elements/Grid"
 import { Separator } from "Styleguide/Elements/Separator"
 import { Spacer } from "Styleguide/Elements/Spacer"
+import { ResponsiveProvider } from "Styleguide/Utils/Responsive"
 import { Articles } from "./Articles"
 import { ArtistHeader } from "./ArtistHeader"
 import { AuctionResults } from "./AuctionResults"
 import { CV } from "./CV"
 import { Overview } from "./Overview"
 import { RelatedArtists } from "./RelatedArtists"
 import { Shows } from "./Shows"
-import { Footer } from "Styleguide/Components/Footer"
+
+injectGlobalCSS()
 
 export class Artist extends React.Component {
   render() {
     return (
-      <Grid fluid>
-        <Row>
-          <Col>
-            <ArtistHeader />
-          </Col>
-        </Row>
+      <Theme>
+        <GlobalStyles>
+          <ResponsiveProvider breakpoints={themeProps.mediaQueries}>
```

Comment:
For stuff under root `/Pages` I've pulled in our providers since these pages will likely be rendered wholesale like this in Force. See [here](https://github.com/artsy/force/pull/2621/files#diff-efaf83a3a5ab87f1c6b9501f8e3dbb10R12) for an example. (Note that we still need to start defining overall routing structure, too. I think I'm going to work on that next)